OBSERVABILITY.md

```diff
@@ -0,0 +1,119 @@
+## Observability with OpenTelemetry
+
+This Cloud Spanner client supports [OpenTelemetry Traces](https://opentelemetry.io/), which gives insight into the client internals and aids in debugging/troubleshooting production issues.
+
+By default, the functionality is disabled. You shall need to add OpenTelemetry dependencies, and must configure and
+enable OpenTelemetry with appropriate exporters at the startup of your application:
+
+**Table of contents:**
+
+* [Observability](#observability)
+  * [Tracing](#tracing)
+    * [OpenTelemetry Dependencies](#opentelemetry-dependencies)
+    * [OpenTelemetry Configuration](#opentelemetry-configuration)
+    * [SQL Statement span annotation](#sql-statement-span-annotation)
+    * [OpenTelemetry gRCP instrumentation](#opentelemetry-grpc-instrumentation)
+    * [Tracing Sample](#tracing-sample)
+
+### Tracing
+
+#### OpenTelemetry Dependencies
+
+Use a trace exporter of your choice, such as Google Cloud Trace like below:
+
+```javascript
+const {
+  TraceExporter,
+} = require('@google-cloud/opentelemetry-cloud-trace-exporter');
+const exporter = new TraceExporter();
+```
+
+#### OpenTelemetry Configuration
+
+```javascript
+const {NodeSDK} = require('@opentelemetry/sdk-node');
+const {Resource} = require('@opentelemetry/resources');
+const {
+  NodeTracerProvider,
+  TraceIdRatioBasedSampler,
+} = require('@opentelemetry/sdk-trace-node');
+const {
+  BatchSpanProcessor,
+} = require('@opentelemetry/sdk-trace-base');
+const {
+  SEMRESATTRS_SERVICE_NAME,
+  SEMRESATTRS_SERVICE_VERSION,
+} = require('@opentelemetry/semantic-conventions');
+
+const resource = Resource.default().merge(
+  new Resource({
+    [SEMRESATTRS_SERVICE_NAME]: 'spanner-sample',
+    [SEMRESATTRS_SERVICE_VERSION]: 'v1.0.0', // The version of your app running.,
+  })
+);
+
+// Wire up the OpenTelemetry SDK instance with the exporter and sampler.
+const sdk = new NodeSDK({
+  resource: resource,
+  traceExporter: exporter,
+  // Trace every single request to ensure that we generate
+  // enough traffic for proper examination of traces.
```

Comment:
```suggestion
  // Modify the following line to adjust the sampling rate.
  // It is currently set to 1.0, meaning all requests will be traced.
```

Reply:
In the sample code we used `NodeTracerProvider` instead of NodeSDK. Lets stick to on which we would want to recomment